features/admin.claims.v1/components/edit/local-claim/edit-basic-details-local-claims.tsx

```diff
@@ -225,6 +277,28 @@ export const EditBasicDetailsLocalClaims: FunctionComponent<EditBasicDetailsLoca
      * Update attribute profile states from claims
      */
     useEffect(() => {
+
+        if (claim.canonicalValues && Array.isArray(claim.canonicalValues)) {
```

Comment:
If we cannot ensure that claim object will always have canonicalValues property, let's use optional chaining.

```suggestion
        if (claim?.canonicalValues && Array.isArray(claim.canonicalValues)) {
```

Reply:
Yes, canonicalValues is optional attribute and it will not available if canonicalValues are not configured.
Fixed by - https://github.com/wso2/identity-apps/pull/8111/commits/fc575885da12bc341fd55470310378ecbc3dc6ad